nemo/collections/asr/models/hybrid_rnnt_ctc_bpe_models.py

```diff
@@ -91,6 +91,13 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
 
         super().__init__(cfg=cfg, trainer=trainer)
 
+        loss_name, loss_kwargs = self.extract_rnnt_loss_cfg(self.cfg.get("loss", None))
+
+        if loss_name == 'tdt':
+            self.cfg.decoding.durations = loss_kwargs.durations
+        elif loss_name == 'multiblank_rnnt':
+            self.cfg.decoding.big_blank_durations = loss_kwargs.big_blank_durations
+
```

Comment:
thanks hainan, can you move this to a local function and call. Can you also make sure to add to `hybrid_rnnt_ctc_model.py` file as well.

Reply:
Done. I wrote a function for the logic and called it from the hybrid_rnnt_bpe class, as well as the original rnnt and rnnt_bpe classes. It seems there is no need to change the file hybrid_rnnt_ctc_model.py since it doesn't have a 'change vocab' function (since it's character-based).